index.bs

```diff
@@ -5855,8 +5868,8 @@ To <dfn>get navigable's user prompt handler</dfn> given |type| and |navigable|:
 </div>
 
 <div algorithm>
-The [=remote end event trigger=] is the <dfn export>WebDriver BiDi user prompt
-opened</dfn> steps given |window|, |type|, |message|, and optional |default value|
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi user prompt opened</dfn> steps
+given {{Window}} |window|, string |type|, string |message|, and optional string |default value|
```

Comment:
Whether we use `string` or `text`. The closed variant of this event uses `text`. Given the CDDL definition we probably want to use `text` here as well.

```suggestion
given {{Window}} |window|, string |type|, string |message|, and optional text |default value|
```